Tests/APITesters/AllAPITests/ObjcAPITester/RCPurchasesAPI.m

```diff
@@ -124,6 +124,12 @@ + (void)checkAPI {
     [p setKeyword: @""];
     [p setCreative: nil];
     [p setCreative: @""];
+    #if !ENABLE_CUSTOM_ENTITLEMENT_COMPUTATION
+    if (@available(iOS 18.0, macOS 15.0, tvOS 18.0, watchOS 11.0, visionOS 2.0, *)) {
```

Comment:
Is the `@available` `if` really needed here?

Reply:
Great callout! I think I'm too used to doing this for win-back offers 😅 Removed the check in a379af34f0dd7e4ead1b604a724253905826aa77 👍